openshift/metabase/oc_build_metabase.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -l
+#
+# Script based on instruction found in readme of
+# https://github.com/bcgov/nr-showcase-devops-tools/tree/master/tools/metabase
+
+export BASE_URL="https://raw.githubusercontent.com/bcgov/nr-showcase-devops-tools/master/tools/metabase/openshift"
```

Comment:
Let's copy / augment this yaml file we're pulling here, it can change on us / get deleted otherwise.